pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2020,12 +2065,18 @@ type RestoreSpec struct {
 	To *TiDBAccessConfig `json:"to,omitempty"`
 	// Type is the backup type for tidb cluster.
 	Type BackupType `json:"backupType,omitempty"`
+	// RestoreMode is the restore mode. such as snapshot or pitr.
+	RestoreMode RestoreMode `json:"restoreMode,omitempty"`
```

Comment:
If you name BackupMode in Backup.Spec as Mode, please do the same thing here, just keep the same style

Reply:
ok